styles.css

```diff
@@ -108,10 +112,1448 @@ body {
 }
 
 .headline-4-ul {
-  list-style: none;
   width: 10.636rem;
   height: 1.25rem;
   display: flex;
   flex-direction: row;
   justify-content: space-between;
 }
+
+/* works */
+.works {
+  display: grid;
+  grid-template-rows: auto;
+  align-items: center;
+  width: 100vw;
+  height: 439.05vh;
+  gap: 5.5rem;
+  padding: 7.125rem 1.5rem;
+}
+
```

Comment:
- [ ] for the best UX/UI kindly fix the overflowing in your website to prevent scrolling left & right 🙏 

![over](https://user-images.githubusercontent.com/89282221/219623979-61683189-e5ac-4a96-aae9-bbe1e6c50ab8.gif)